elements/dash-tooltip/demo.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+
+<style>
+  /* When the tooltip is hidden, it shouldn't be visible. */
+  dash-tooltip[aria-hidden="true"] {
+    display: none !important;
+  }
+
+  /* The tooltip is by default unstyled. */
+  dash-tooltip {
+    margin-left: 8px;
+    padding: 4px;
+    background-color: #616161;
+    color: #fff;
+    border-radius: 3px;
+    opacity: 0.9;
+
+    /* Optionally, you can animate the tooltip. */
+    -webkit-animation: fadein 1s;
+    animation: fadein 1s;
+  }
+
+  @keyframes fadein {
+    from { opacity: 0; }
+    to   { opacity: 1; }
+  }
+</style>
+
+<div class="text">
+   <label id="lb1" for="name">Your name:</label>
+   <input id="name" aria-labelledby="lb1" aria-describedby="tp1"/>
```

Comment:
Why're you using `aria-labelledby` if you are also using the `for` attribute of `<label>`?